app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3275,19 +3256,18 @@ class PatientRepositoryAndroidTest {
         streetAddress = userEnteredPatientStreetAddress,
         zone = userEnteredPatientZone
     )
-    patientRepository.saveOngoingEntry(ongoingPatientEntry)
 
     // when
     val savedPatient = patientRepository
         .saveOngoingEntryAsPatient(
+            patientEntry = ongoingPatientEntry,
             loggedInUser = loggedInUser,
             facility = facilityToSavePatientAt,
             patientUuid = UUID.fromString("0d8b5e31-2695-49b3-ba23-d4f198012870"),
             addressUuid = UUID.fromString("05dfcc13-b855-4e06-be91-7c09c79d98ca"),
             supplyUuidForBpPassport = { UUID.fromString("b6915d20-d396-4c5d-b25c-4b2d9b81fd2b") },
-            supplyUuidForAlternativeId = { UUID.fromString("91539c5f-70a0-4e69-9740-fa8b37fa2f16") },
-            supplyUuidForPhoneNumber = { UUID.fromString("b842da76-26f8-4d7d-814a-415209335ecb") }
-        )
+            supplyUuidForAlternativeId = { UUID.fromString("91539c5f-70a0-4e69-9740-fa8b37fa2f16") }
+        ) { UUID.fromString("b842da76-26f8-4d7d-814a-415209335ecb") }
```

Comment:
Can you tell me what is this UUID string for? phone number UUID?

Reply:
Yes, I don't know why but it seems like the IDE auto format is moving the last lambda parameter out of the method invocation automatically. I'll fix this in the next PR in the refactoring work.